requirements.txt

```diff
@@ -20,7 +20,8 @@ twine==3.4.1
 wheel==0.36.2
 
 # Tests & Linting
-anyio==3.1.0
+# https://github.com/agronholm/anyio/pull/309
+anyio @ git+https://github.com/agronholm/anyio.git@9ae936079d407223a899c635b6cc859a64bc0124
```

Comment:
Okay, let's consider this change as being blocked on an `anyio` 3.2 release, I guess?